README.asciidoc

```diff
@@ -339,13 +338,13 @@ directory. It will first look for an `autoload` directory at
 it does not exist.
 
 Once all those files are loaded, Kakoune will try to source
-`${runtime}/kakrc.local` which is expected to contain distribution provided
+`${runtime}/kakrc.local`, which is expected to contain distribution provided
 configuration.
 
-And finally, the user configuration will be loaded from `${userconf}/kakrc`.
+Finally, the user configuration will load from `${userconf}/kakrc`.
 
 NOTE: If you create a user `autoload` directory in `${userconf}/autoload`,
-the system one at `${runtime}/autoload` will not be loaded anymore. You can
+the system one at `${runtime}/autoload` will not load anymore. You can
```

Comment:
Not sure about "loads" vs. "is loaded" (same above).
I think of the editor as the subject ("the editor loads the configuration file") that's why passive voice makes more sense to me.

Reply:
I had thought that as well. There's a terrible tendency for people to use the passive in contexts that don't warrant it. However, in this case, I think you're right. I'll change it back.